src/condor_tools/htcondor_cli/annex.py

```diff
@@ -93,7 +93,20 @@ class Create(Verb):
             "help": "Location to store temporary annex control files, probably should not be changed. Defaults to %(default)s",
             "type": Path,
             "default": Path(htcondor.param.get("ANNEX_TMP_DIR", "~/.hpc-annex")),
-        }
+        },
+        "cpus": {
+            "args": ("--cpus",),
+            "help": "Number of CPUs to request (shared queues only).  Unset by default.",
+            "type": int,
+            "default": None,
+        },
+        "mem_mb": {
+            "args": ("--mem_mb",),
+            # TODO: Parse units instead of requiring this to be a number of MBs
+            "help": "Memory (in MB) to request (shared queues only).  Unset by default.",
+            "type": int,
+            "default": None,
```

Comment:
Kind of hate that specifying these does nothing for some annexes. Hopefully `shared queues only` is enough of an explanation.

Reply:
I moved my rant from this comment into a [ticket](https://opensciencegrid.atlassian.net/browse/HTCONDOR-1018).